README

```diff
@@ -27,6 +27,10 @@ The basic features of Geany are:
 - embedded terminal emulation
 - extensibility through plugins
 
+Changelog
+---------
+See the release notes on `the website <https://www.geany.org/documentation/releasenotes/>`_ or `on GitHub <https://github.com/geany/geany/releases>`_.
```

Comment:
ReleaseNotes and ChangeLog are not the same. The ChangeLog is a generated list from the GIT history, i.e. it contains each commit.
The ReleaseNotes are hand-written highlight changes for each release and much shorter.

IMO nowadays, most people look for what we put in ReleaseNotes and those who want to check every single commit, probably already know how to find the GIT history.